expat/lib/xmlparse.c

```diff
@@ -1119,10 +1126,12 @@ XML_SetEncoding(XML_Parser parser, const XML_Char *encodingName)
   */
   if (ps_parsing == XML_PARSING || ps_parsing == XML_SUSPENDED)
     return XML_STATUS_ERROR;
-  if (encodingName == NULL)
+  FREE((void *)protocolEncodingName);
```

Comment:
So this is about the previous one. Let's add a blank line after to separate from what's coming and maybe also a blank line before (and a comment that this is about previous, while the code after is about the new one).